runtime/src/bank.rs

```diff
@@ -6853,13 +6867,14 @@ impl Bank {
         }
 
         info!(
-            "bank frozen: {} hash: {} accounts_delta: {} signature_count: {} last_blockhash: {} capitalization: {}",
+            "bank frozen: {} hash: {} accounts_delta: {} signature_count: {} last_blockhash: {} capitalization: {}, epoch_accounts_hash: {:?}",
             self.slot(),
             hash,
             accounts_delta_hash.hash,
             self.signature_count(),
             self.last_blockhash(),
             self.capitalization(),
+            epoch_accounts_hash,
```

Comment:
I think I'd rather see a separate info that logs this only when we included the hash. bank frozen is already a very common info I search for. Making it longer with a None is not helpful ;-)